src/app_service/service/transaction/service.nim

```diff
@@ -232,14 +234,19 @@ QtObject:
       hopTx.amount = route.amountIn.some
       hopTx.bonderFee = route.bonderFees.some
       path.hopTx = hopTx
+    elif(route.bridgeName == ERC721_TRANSFER_NAME):
+      eRC721TransferTx = txData
+      eRC721TransferTx.chainID =  route.toNetwork.chainId.some
+      eRC721TransferTx.recipient = parseAddress(to_addr).some
+      eRC721TransferTx.tokenID = parseInt(tokenSymbol).some
```

Comment:
we shouldn't use `parseInt` and type `int` for this. TokenID is a `UInt256` variable, we should treat it like we treat the `value` field.

Reply:
done @dlipicar !